Jenkinsfile-build

```diff
@@ -0,0 +1,68 @@
+def versionTag = "v${env.BUILD_NUMBER}"
+
+pipeline {
+    agent none
+
+    stages {
+      stage('Build keycloak') {
+        agent {
+          label 'master'
+        }
+        when {
+          expression {
+            env.BRANCH_NAME == "master"
```

Comment:
Are there any build steps we want to run when the branch isn't master? e.g. would it be helpful to run the build steps without pushing, to catch some errors before the branch is merged?

If not, maybe this should just be a regular pipeline build rather than a multibranch pipeline (since if you're going to filter it to just one branch, you don't need all the multibranch scanning).

Reply:
Build the docker image first on any branch and build / push only on the master branch